packages/editor/src/components/EventHandler.tsx

```diff
@@ -47,7 +48,7 @@ const EventHandler = ({ pubSub, tab }) => {
       id: tab.id,
       projectId: config.projectId,
     })
-  }, [config.projectId, getSpell, tab.id, tab.name])
+  }, [config.projectId, getSpell, tab.id, tab.name, tab])
```

Comment:
> if `tab.id` and `tab.name` are being checked here already, do we need `tab`?
> 
> And if we do need `tab` here, then can we remove `tab.id` and/or `tab.name`

We do not need `tab.id` and `tab.name` since they are not triggering useEffect when tab changed, so `tab` is enough

Reply:
> if `tab.id` and `tab.name` are being checked here already, do we need `tab`?
> 
> And if we do need `tab` here, then can we remove `tab.id` and/or `tab.name`

We do not need `tab.id` and `tab.name` since they are not triggering useEffect when tab changed, so `tab` is enough